Scripts/Runtime/Entities/Data/IWorldUniqueID.cs

```diff
@@ -3,13 +3,17 @@
 namespace Anvil.Unity.DOTS.Entities
 {
     /// <summary>
-    /// Provides a <see cref="World"/> unique ID
+    /// Provides a <see cref="World"/> unique ID.
+    /// This ID is unique within the current World instance.
+    /// This ID is not unique globally between all world instances.
```

Comment:
I would be more explicit here:

```suggestion
    /// This ID is guaranteed to be the same across all world instances.
```